src/components/FriendList/FriendListHome.js

```diff
@@ -0,0 +1,134 @@
+import NavDrawer from "../NavDrawer/navDrawer";
+import {BackgroundCard, StudyRoomChatCard} from "../CustomMUIComponents/CustomCards";
+import * as React from "react";
+import {Stack} from "@mui/material";
+import {PrimaryButton2} from "../CustomMUIComponents/CustomButtons";
+import BottomDrawer from "../StudyRoom/BottomDrawer";
+import PersonSearchIcon from '@mui/icons-material/PersonSearch';
+import {StudyRoomCard} from "../StudyRoom/CommonResources";
+import Button from "@mui/material/Button";
+import ClearIcon from '@mui/icons-material/Clear';
+import Typography from "@mui/material/Typography";
+import FriendSearch from "./FriendSearch";
+import Badge from '@mui/material/Badge';
+import axios from "axios";
+import {useCallback, useState} from "react";
+import FriendNotification from "./FriendsNotification";
+import GroupIcon from '@mui/icons-material/Group';
+
+
+export default function FriendListHome(){
+    const [loading,setLoading] = useState(true);
+    const [friends, setFriends] = React.useState([]);
+    const [visible, setVisible] = React.useState(false);
+    const email = JSON.parse(localStorage.getItem("email"));
```

Comment:
instead of calling email like that, you can use the GetAuthentication method in the authentication component. it will return you email, username and token, if it exists, will redirect the user to login otherwise